ROADMAP.md

```diff
@@ -1,25 +1,389 @@
+
+Sigstore is inherently flexible and can natively support a variety of:
+
+* deployment scenarios (e.g. public-facing VS private enterprise deployments), 
+* key management approaches (e.g. hardware-backed keys stored in HSMs or Yubikeys or ephemeral scenarios where a private key can be deleted immediately after use),
+* operating environments (public or private clouds, on-premises),
+* identities provided by OpenID Connect tokens, including developer and workload identities
+* programming languages
+
+Since its founding, Sigstore has grown into one of the top 25 largest and fastest growing open source communities under the Linux Foundation. The community includes over 1,400 unique contributors composed of both unaffiliated individuals and 91 unique organizations. Sigstore projects and services are sponsored by the Open Source Security Foundation (OpenSSF) which provides neutral governance and oversight to the project. 
+
+Sigstore is best thought of as an “umbrella brand”, encompassing [several open source projects](https://github.com/sigstore) and public-good services which are provided to everyone at no cost. These instances aim to adhere to a 99.5% SLO and are supported by a multi-vendor on-call rotation to ensure neutrality and a tight feedback loop between project maintainers and service operators.
+
+## Current Status (as of September 2023)
+
+At the time of writing, the public Rekor signature transparency log contains over 32 million entries representing signatures from more than 17,000 unique open source projects including Kubernetes, CPython, Istio, urllib3, and Tekton (more [adopters](https://github.com/sigstore/friends) can be seen at the Sigstore [project landscape](https://landscape.openssf.org/sigstore)). Sigstore’s certificate authority has issued more than 21 million short-lived code signing certificates based on the OpenID Connect identity binding approach.  
+
+Sigstore announced the [general availability](https://openssf.org/press-release/2022/10/25/sigstore-announces-general-availability-at-sigstorecon/) of its public-good services in October of 2022 which built upon the v1.0 releases of Cosign, Fulcio, and Rekor. The public services have consistently met their 99.5% availability SLO since that milestone.
+
+Sigstore has several implementations of its core signing and verification workflows for popular programming languages, including:
+
+| Programming Language | Repository               |       Status      |
+|:--------------------:|--------------------------|:-----------------:|
+|        Golang        | [sigstore/cosign](https://github.com/sigstore/cosign)          |         GA        |
```

Comment:
Imo it's best to refer to the Go programming language as "Go" and leave "Golang" for use as a search term. 😄